src/databricks/labs/ucx/install.py

```diff
@@ -227,12 +227,14 @@ def _is_testing(self):
     def _prompt_for_new_installation(self) -> WorkspaceConfig:
         logger.info("Please answer a couple of questions to configure Unity Catalog migration")
         default_database = "ucx"
+        default_catalog = "hive_metastore"
         # if a workspace is configured to use external hive metastore, the majority of the time that metastore will be
         # shared with other workspaces. we need to add the suffix to ensure uniqueness of the inventory database
         if self.policy_installer.has_ext_hms():
             default_database = f"ucx_{self.workspace_client.get_workspace_id()}"
+        inventory_catalog = self.prompts.question("Inventory Catalog", default=default_catalog, valid_regex=r"^\w+$")
```

Comment:
I do not think we want to do this. We still want the inventory to be in the `hive_metastore` (always), but need the drop in for the catalog to built the history table #2572